sqladmin/models.py

```diff
@@ -688,6 +688,8 @@ def __init__(self) -> None:
         self._custom_actions_in_detail: Dict[str, str] = {}
         self._custom_actions_confirmation: Dict[str, str] = {}
 
+        self.page_size_ = self.page_size
```

Comment:
I think this is a bit unclear, maybe it's better to have a method `def get_page_size` instead?

Reply:
I think you should get rid of `self._page_size` as we already have a `self.page_size`. In order to get rid of that I think a helper method will be useful.

Reply:
I think , defining any method,we need variable to track changeable value of pagesize.
but, What variable do you suggest for that?
In above scenario, by replaceing self._page_size with something else will resolve the case.

Reply:
I don't think we have to store it in a variable? Just as it was previously working it's either the current value coming from the request parameter or the default self.page_size.

Reply:
In list.html line_number  (197)
 {{ request.query_params.get("pageSize") or model_view.page_size }} / Page
 
when we leave as it is,
on passing pagesize=somevalue
domainname/admin/address/list?pageSize=somevalue
 
 this somevalue is displayed like
 ![pagesize](https://github.com/aminalaee/sqladmin/assets/46397975/88b23785-b83a-4980-ad6c-93005c9fa07d)

Reply:
yeah but you could call`{{ model_view.validate_page_size(request.query_params.get("pageSize")) or model_view.page_size }} / Page` right?

Reply:
Yes,
we can do that, as well we have to set default value and change it's structure like

 model_view.validate_page_size((request.query_params.get("pageSize")) or model_view.page_size),model_view.page_size)
 
but it looks quite messy

Reply:
I think it's better than having `self.page_size` and `self.page_size_` and also the page size should be evaluated per request, we can't store the current page size on model_view. The `page_size` stored is the configuration which doesn't change per request, we shouldn't store per request page_size there.

Reply:
in list.html
          {{ model_view.validate_page_number((request.query_params.get("pageSize") or model_view.page_size),model_view.page_size)  }} / Page

in models.py
def validate_page_number(self, page_value: int|str, default: int) -> int:
        try:
            return int(page_value)
        except ValueError:
            return default
  
async def list(self, request: Request) -> Pagination:
        page = max(1,self.validate_page_number(request.query_params.get("page", 1), 1))
        page_size = self.validate_page_number(request.query_params.get("pageSize", 0), 0)
        
 After updating  , it looks like that
 If all the changes looks good to you
 I will commit the changes